gtsam/geometry/Gal3.h

```diff
@@ -217,6 +219,34 @@ class GTSAM_EXPORT Gal3 : public MatrixLieGroup<Gal3, 10, 5> {
   static TangentVector Vee(const LieAlgebra& X);
 
   /// @}
+  /// @name Dynamics
+  /// @{
+
+  // φ: autonomous flow where velocity acts on position for
```

Comment:
I think this is not needed for Gal3. The autonomous flow is implemented by the group product. That's what makes it so great :-)